addons/survey/wizard/survey_invite.py

```diff
@@ -151,10 +152,16 @@ def create(self, vals_list):
     @api.depends('template_id', 'partner_ids')
     def _compute_subject(self):
         for invite in self:
-            langs = set(invite.partner_ids.mapped('lang')) - {False}
-            if len(langs) == 1:
-                invite = invite.with_context(lang=langs.pop())
-            super(SurveyInvite, invite)._compute_subject()
+            if invite.subject:
+                return
+            else:
+                langs = set(invite.partner_ids.mapped('lang')) - {False}
+                if len(langs) == 1:
+                    invite = invite.with_context(lang=langs.pop())
+                super(SurveyInvite, invite)._compute_subject()
```

Comment:
We don't have to call super, do we ?

Reply:
We have to call super because we are using a mail template, meaning that if it is not called then the template will not be computed for the subject, leaving it blank.

Reply:
Oh yes, my bad. It is done in two steps, fetching template value into invite wizard field then rendering it. I read it a bit too quickly :)